examples/basic/basic.c

```diff
@@ -22,7 +22,7 @@ int main(void) {
     // Allocations are made with mmap. The default values of params result
     // in an mmap call like this:
     // mmap(NULL, size, PROT_READ | PROT_WRITE, MAP_ANONYMOUS | MAP_PRIVATE, -1, 0)
-    umf_memory_provider_ops_t *provider_ops = umfOsMemoryProviderOps();
+    const umf_memory_provider_ops_t *provider_ops = umfOsMemoryProviderOps();
```

Comment:
compat tests should fail here - I need to investigate this

Reply:
hm I see only warnings: https://github.com/oneapi-src/unified-memory-framework/actions/runs/14404103366/job/40397848766?pr=1257